pkg/apis/extensions/types.go

```diff
@@ -82,43 +81,44 @@ type SubresourceReference struct {
 	Subresource string `json:"subresource,omitempty"`
 }
 
-// ResourceConsumption is an object for specifying average resource consumption of a particular resource.
-type ResourceConsumption struct {
-	Resource api.ResourceName  `json:"resource,omitempty"`
-	Quantity resource.Quantity `json:"quantity,omitempty"`
+type CPUTargetUtilization struct {
+	// Utilization target percent defines fraction of the requested CPU that should be utilized/used,
+	// e.g. 70 means that 70% of the requested CPU should be in use.
+	TargetPercent uint `json:"targetPercent"`
```

Comment:
So far we haven't used uint anywhere in the API. It's not supported by all client languages. Please just use int.

Also, please use Percentage rather than Percent.